deepchecks/tabular/dataset.py

```diff
@@ -546,6 +546,13 @@ def train_test_split(self: TDataset,
     def _infer_label_type(label_col: pd.Series):
         if is_categorical(label_col, max_categorical_ratio=0.05):
             if label_col.nunique(dropna=True) > 2:
+                if infer_dtype(label_col) == 'integer' \
+                        and label_col.nunique() > 10:
```

Comment:
I think that 10 is too large. There's an issue here as some problems (discrete ordinal features) can actually be tackled both as regression and as multiclass (e.g. 1-5 stars in reviews).
So maybe >=5?

Reply:
sure